### PR TITLE
feat: create player settings page

### PR DESCRIPTION
**Pull request by ferferga**
Cherrypicked from #633 

This might not work standalone or look right as it was made around that PR. That PR must be merged first.

### CODE REVIEW

File: pages/settings/admin/players.vue [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+        crossfade (or other incompatible settings with the platform) are enabled.
+        Ellaborate also a way to generate this incompatibilities programatically -->
+      <v-slide-y-transition>
+        <v-alert v-if="airplay" type="warning" outlined prominent border="left">
+          {{ $t('settings.players.disabledFeatures') }}
+          <br />
+          <br />
+          <strong> · Crossfade</strong>
+        </v-alert>
+      </v-slide-y-transition>
```
Comment (Camc314) [inline pages/settings/admin/players.vue:32]: I don't think we should have settings for things that aren't yet implemented

File: pages/settings/admin/players.vue [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+      </v-row>
+      <!-- TODO: Bind this to user's settings, so this warning is only shown when
+        crossfade (or other incompatible settings with the platform) are enabled.
+        Ellaborate also a way to generate this incompatibilities programatically -->
+      <v-slide-y-transition>
+        <v-alert v-if="gcast" type="warning" outlined prominent border="left">
+          {{ $t('settings.players.disabledFeatures') }}
+          <br />
+          <br />
+          <strong> · Crossfade</strong>
```
Comment (Camc314) [inline pages/settings/admin/players.vue:60]: Two crossfades?

**Comment by ferferga** [comment]
I agree with you, this page was made mainly as a PoC of contextual alerts based on selectable settings, (picture from #633):



You can see the main post video too to check my proposal for them: basically an alert that appears when you interact with settings in a contextual way based on your other settings.

We can leave this open until the features start to get implemented. I already investigated a bit about Chromecast support so it shouldn't take long, but def not before the preview release.

File: pages/settings/admin/players.vue (hunk 2 above)
Comment (ferferga) [reply pages/settings/admin/players.vue:60]: See my comment below

**Comment by ferferga** [comment]
@jellyfin rebase

**Pull request closed without merging**